### ISSUE TITLE
Add  ability to make windows curved to face user.

### ISSUE THREAD

**Issue by StarrKiss** [SimulaVR/Simula#117]
The Non FOSS project 'Immersed', which has pricing, has the windows curved like a curved monitor to face the user, as can be seen here- https://immersedvr.com/
It would be great if this too could implement curved windows so that it is easier to view.

**Comment by thorsten-passfeld** [comment]
Yes please. That would be an awesome feature. I hope this gets implemented.
